indico/web/client/js/react/components/principals/Search.jsx

```diff
@@ -58,38 +59,48 @@ const searchFactory = config => {
   } = config;
 
   /* eslint-disable react/prop-types */
-  const FavoriteItem = ({name, detail, added, onAdd, onRemove}) => (
-    <Dropdown.Item
-      disabled={added}
-      styleName="favorite"
-      style={{pointerEvents: 'all'}}
-      onClick={e => {
-        e.stopPropagation();
-        if (added) {
-          onRemove();
-        } else {
-          onAdd();
-        }
-      }}
-    >
-      <div styleName="item">
-        <div styleName="icon">
-          <Icon.Group size="large">
-            <Icon name={resultIcon} />
-            {added && <Icon name="check" color="green" corner />}
-          </Icon.Group>
-        </div>
-        <div styleName="content">
-          <List.Content>{name}</List.Content>
-          {detail && (
-            <List.Description>
-              <small>{detail}</small>
-            </List.Description>
-          )}
+  const FavoriteItem = ({name, detail, added, onAdd, onRemove, avatarURL}) => {
+    const avatar = avatarURL ? (
+      <Image src={avatarURL} size="mini" avatar />
+    ) : (
+      <Icon name={resultIcon} />
+    );
+
+    const marginLeft = avatarURL ? 'margin-left' : '';
```

Comment:
I don't think that's a good name for a CSS class. I'd try to find a name that describes the meaning, e.g. `has-avatar`.

Reply:
I used `has-avatar`